### DISCUSSION TITLE
按照官方文档引入的友链信息，样式异常

### DISCUSSION THREAD

**Discussion by chendada00** [Qexo/Qexo#546] [Q&A]
那位大哥帮忙看下这是什么问题，是因为css冲突？
![image](https://github.com/Qexo/Qexo/assets/53457984/e114373a-266b-49b1-90cf-fb6963e35d6e)
![image](https://github.com/Qexo/Qexo/assets/53457984/06861b82-059e-4bdb-9abd-cbfc07f32be1)
![image](https://github.com/Qexo/Qexo/assets/53457984/4fa20760-1fbe-4f0c-a0f0-5714d7dacf1b)

**Comment by Fgaoxing** [comment]
这个是没有进行主题适配的默认版本，你给我网站，我写个适配版本

**Reply by chendada00** [reply]
我的网站是 ctext.top
主题是 anzhiyu

**Comment by Fgaoxing** [comment]
anzhiyu的分支太多了，你说anzhiyu我很难确定其实，我一会看看

**Comment by Fgaoxing** [comment]
我按照这个写吧https://blog.zhheo.com/link/

**Comment by Fgaoxing** [comment]
估计需要明天

**Reply by chendada00** [reply]
好的好的，感谢

**Comment by Fgaoxing** [comment]
抱歉忙完了，现在搞

**Reply by chendada00** [reply]
老哥弄完了么

**Reply by Fgaoxing** [reply]
没有，懒加载的问题，我研究研究吧
